test/gtest/db_sync.cpp

```diff
@@ -547,21 +586,55 @@ TEST(DbSync, DISABLED_StaticFDBSync)
                        pdb_vals.end())  << "Non-Tunable solver found in PDB" << solv.GetSolverDbId() ;
             ++fdb_idx;
         }
-
-        std::cout << "Lines of find db completed:" << ++cnt_finddb_entry << std::endl;
+        if(kidx % 100 == 0)
+            std::cout << "Lines of find db completed:" << kidx << std::endl;
+        counter.fetch_add(1, std::memory_order_relaxed);
     }
+}
 
-    // for each key , val pair in the find db
-    // for each solver entry in the val of find db
-    // check that the solver is applicable
-    // check the workspace value is correct
-    // Make sure the perf db entry is valid
-    // make sure the kdb entry exists and is loadable in hip
-    // Bonus: No phantom entries in perf db, no phantom entries in kdb
+void StaticFDBSync(const std::string& arch, const size_t num_cu)
+{
+    boost::filesystem::path fdb_file_path, pdb_file_path, kdb_file_path;
+    auto& handle = get_handle();
+    if(handle.GetDeviceName() != arch)
+        GTEST_SKIP();
+    setenv("MIOPEN_DEVICE_CU", std::to_string(num_cu).c_str(), 1);
```

Comment:
Setting the env this way doesn't achieve the desired result.
For 90a 104 / 110, there is no cu change and the same test runs 2x.